content/en/guide/v10/typescript.md

```diff
@@ -170,7 +170,7 @@ class Expandable extends Component<ExpandableProps, ExpandableState> {
 
 ## Typing events
 
-Preact emits regular DOM events. As long as your TypeScript project includes the `dom` library (set it in `tsconfig.json`), you have access to all event types that are available in your current configuration.
```

Comment:
> In Preact core, `onChange` is the standard [DOM change event]

There is "standard DOM event" in other page.
Therefore, I change to "standard DOM event".
Element emits event.